xdsl/ir/core.py

```diff
@@ -708,6 +708,33 @@ def __hash__(self):
         return hash(self._op._operands)  # pyright: ignore[reportPrivateUsage]
 
 
+class OpTraits(Iterable[OpTrait]):
```

Comment:
Can you add one bit of documentation on the class?

Reply:
Hope it's roughly the documentation you were hoping for